create-project.sh

```
@@ -0,0 +1,64 @@
+
+# Copy template
+echo "📁 Copying template..."
+cp -r . "$PROJECT_DIR"
+
+# Remove git history
+echo "🔄 Removing template git history..."
+rm -rf "$PROJECT_DIR/.git"
+
+# Update package.json
+echo "📝 Updating package.json..."
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)